cmd/legacy_param_mapper_test.go

```diff
@@ -505,6 +505,80 @@ func TestInvalidClientProtocol(t *testing.T) {
 	assert.NotNil(t, err)
 }
 
+func TestLogSeverityRationalization(t *testing.T) {
```

Comment:
Nit: I think this unit test is not required. WDYT?

Reply:
It helps validate that Rationalize is being called in the mapper, although it doesn't need to be this exhaustive. In any case, no harm no foul.